### PR TITLE
Vel/python/lab09

### PR DESCRIPTION
**Pull request by Velshaun**

### CODE REVIEW

**Review by chapman311 — CHANGES_REQUESTED** [review]
Output does not give what Lab 9 asks for. Please look back at the lab instructions.

**Comment by Velshaun** [comment]
I was returning the list item instead of the index. Problem solved!

**Review by chapman311 — APPROVED** [review]
Code looks great. I really like the protection from looping over the 0 and last index positions in your for loops. Good job with the change to record the index position too!

File: code/vel/python/lab09_ver1.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+data = [1, 2, 3, 4, 5, 6, 7, 6, 5, 4, 5, 6, 7, 8, 9, 8, 7, 6, 7, 8, 9]
+
+def find_peaks():
+    peaks = []
+    for i in range(1, len(data)-1): # range(start, stop, step) start = 1
+        if data[i -1] < data[i] and data[i + 1] < data[i]:
+            peaks.append(i)
+    return peaks
+
+#IndexError index out of range
```
Comment (chapman311) [inline code/vel/python/lab09_ver1.py:13]: In the future, try to clean up these kinds of troubleshooting/coding comments before submitting.

**Review by pjz987 — APPROVED** [review]

**Pull request merged**